plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -151,11 +151,9 @@ export const ApiExplorerTable = ({
 
   if (error) {
     return (
-      <div>
-        <Alert severity="error">
-          Error encountered while fetching catalog entities. {error.toString()}
-        </Alert>
-      </div>
+      <WarningPanel severity="error" title="Could not fetch catalog entities.">
+        {error.toString()}
```

Comment:
How about using a code block for the error messages?

Reply:
Good idea. Funny, initially I was going to wrap the entire expandable children area (the "white box") of the new `<WarningPanel>` with a code block, but then realized the children could be of any format and there could be a lot more value in letting someone design more advanced error msgs.

So just added code fences to 3 of the 4, as the 4th Lighthouse doesn't return error traces or HTTP codes, and instead just a sentence for now. Can always be customized later!